library/Ivoz/Provider/Domain/Model/DestinationRate/DestinationRateDto.php

```diff
@@ -34,6 +34,21 @@ public static function getPropertyMap(string $context = '', string $role = null)
             ];
         }
 
+        if ($context === self::CONTEXT_DETAILED) {
+            return [
+                'cost' => 'cost',
+                'connectFee' => 'connectFee',
+                'rateIncrement' => 'rateIncrement',
+                'groupIntervalStart' => 'groupIntervalStart',
+                'id' => 'id',
+                'destinationRateGroupId' => 'destinationRateGroup',
+                'destinationId' => 'destination',
+                'tpRateId' => 'tpRate',
+                'tpDestinationRateId' => 'tpDestinationRate',
```

Comment:
This fields are missing in getDestinationRate.feature

Reply:
These fields are not being used, but it was there before this new feature, what should I do?

- [ ] Delete it.
- [ ] Set them in the Assembler.